### PR TITLE
chore(dynamodb) add template generator

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

add template generator for dynamoDB connector

### CODE REVIEW

File: connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/AwsDynamoDbOperationFactory.java [hunk 1]
```diff
@@ -35,24 +35,17 @@ public static AwsDynamoDbOperationFactory getInstance() {
...
-    } else if (input instanceof GetItem) {
-      return new GetItemOperation((GetItem) input);
-    } else if (input instanceof ScanTable) {
-      return new ScanTableOperation((ScanTable) input);
-    } else if (input instanceof UpdateItem) {
-      return new UpdateItemOperation((UpdateItem) input);
-    } else {
-      throw new UnsupportedOperationException("Unsupported operation : [" + input.getClass() + "]");
-    }
+    return switch (input) {
```
Comment (sbuettner) [inline connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/AwsDynamoDbOperationFactory.java:38]: Love that we are using the new switch pattern matching here! 🥳

File: connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/model/AwsInputDeserializer.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+   */
+  @Override
+  public AwsInput deserialize(JsonParser jp, DeserializationContext ctxt)
+      throws IOException, JsonProcessingException {
+    JsonNode node = jp.getCodec().readTree(jp);
+
+    var operationType = determineActionType(node);
+    ObjectMapper mapper = (ObjectMapper) jp.getCodec();
+    return switch (operationType) {
+      case "createTable" -> mapper.convertValue(node, CreateTable.class);
```
Comment (igpetrov) [inline connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/model/AwsInputDeserializer.java:45]: Could be nice if these literals would be constants and shared/reused in related places like `@TemplateSubType(id = CONSTANT)`

File: connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/model/TableInput.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import io.camunda.connector.generator.java.annotation.TemplateDiscriminatorProperty;
+import io.camunda.connector.generator.java.annotation.TemplateSubType;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonDeserialize()
+@TemplateSubType(label = "Table", id = "tableOperation")
+@TemplateDiscriminatorProperty(
+    name = "tableOperation",
+    group = "operation",
+    label = "Select Action")
```
Comment (igpetrov) [inline connectors/aws/aws-dynamodb/src/main/java/io/camunda/connector/aws/dynamodb/model/TableInput.java:20]:
```suggestion
    label = "Select action")
```

**Review by igpetrov — APPROVED** [review]
LGTM

**Review by igpetrov — APPROVED** [review]

**Pull request merged**